test/kinematics/kinetic_energy_of_body_from_mass_and_velocity_test.py

```diff
@@ -0,0 +1,42 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+
+from symplyphysics import (
+    units, convert_to, SI, errors
+)
+from symplyphysics.laws.kinematics import kinetic_energy_of_body_from_mass_and_velocity as kinetic_energy
+
+@fixture
+def test_args():
+    body_mass = units.Quantity('body_mass')
+    SI.set_quantity_dimension(body_mass, units.mass)
+    SI.set_quantity_scale_factor(body_mass, 0.5 * units.kilogram)
+    body_velocity = units.Quantity('body_velocity')
+    SI.set_quantity_dimension(body_velocity, units.velocity)
+    SI.set_quantity_scale_factor(body_velocity, 0.5 * units.meter / units.second)
+    Args = namedtuple('Args', ['body_mass', 'body_velocity'])
+    return Args(body_mass=body_mass, body_velocity=body_velocity)
```

Comment:
You can use shorter names here, like m and V. It makes tests shorter and does not impact readability of laws.